update the quench values for oatmeal and grits

db48x opened the pull request:
#### Summary
Balance "update the quench values for oatmeal and grits"

#### Purpose of change

Avoid wasting water when cooking oatmeal.

#### Describe the solution

@Papaya-Plan added quench values for oatmeal when he fixed #68313, copying the values from pasta. But when you cook pasta you drain off most of the water, which is very different from oatmeal.

I therefore weighed some oatmeal before and after heating it up. It lost 20 grams out of the 220 grams that I had added to it, meaning it retained 90.45% of the water used to cook it. 50 × 90% = 45.

Also, the mass of the grits was wrong because it left out the mass of the absorbed water. I just assume it’s the same as oatmeal.

#### Describe alternatives you've considered

None; I don’t think there’s any other way to change these values.

#### Testing
![Screenshot from 2023-10-03 00-19-46](https://github.com/CleverRaven/Cataclysm-DDA/assets/228849/9e362e07-a3e6-4b82-9231-3c2c5a67965f)

db48x committed:
update the quench values for oatmeal and grits

I weighed some oatmeal before and after heating it up. It lost 20 grams out of the 220 grams that I had added to it, meaning it retained 90.45% of the water used to cook it. 50 × 90% = 45.

Also, the mass of the grits was wrong because it left out the mass of the absorbed water. I just assume it’s the same as oatmeal.

Calavera20012 commented:
Since most food values are not based in water % (if I'm remembering correctly, I could be wrong) giving these food products what ,essentially ammounts to "how much it quenchs your thirst" similar to those of water when soda is considerably lower (despite a full-calorie soda or carbonated water being typically composed of 90% water, while diet soda is about 99% water​). I think that this, without other changes to food based on water content and possibly looking at the Thirst need, would break the game's balancing a bit.

db48x commented:
You could be right, though I’ll note that oatmeal and grits don’t normally have enough salt or sugar to make you more thirsty. Meanwhile a soda is about 10% sugar by volume, so of course it doesn’t really quench your thirst very well.

Maleclypse commented on data/json/items/comestibles/wheat.json:
```diff
@@ -250,13 +252,14 @@
     "id": "cheese_grits",
     "name": { "str_sp": "cheese grits" },
-    "weight": "80 g",
+    "weight": "306 g",
```
```suggestion
    "weight": "276 g",
```

Maleclypse replied:
Trying to get the density test to pass.

Maleclypse committed:
Update data/json/items/comestibles/wheat.json

db48x committed:
tweak the mass of cheese grits to pass the tests

Really we ought to adjust the volume, but since the recipe for cheese
grits involves adding either a small chunk of hard cheese or a whole
cup(!) of cottage cheese, the stats are going to be pretty whack no
matter what. And let’s face it, nobody keeps enough cheese grits on
hand to make the volume and mass stats meaningful.

db48x commented:
Oops. I just pushed that commit by sheer muscle memory, before I ran the tests :)
